### PR TITLE
WIP kubevirt, presubmit: Add OVN k8s lane

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by oshoval** [comment]
Thanks Brian

/rehearse

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -176,6 +176,74 @@ presubmits:
...
+        - /usr/local/bin/runner.sh
+        - /bin/bash
+        - -ex
+        - |
+          curl -L https://github.com/kubevirt/kubevirtci/pull/1193.patch | git am -3
+          export KUBEVIRT_PROVIDER=kind-ovn
+          make cluster-up
+          export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
+          export DOCKER_PREFIX=localhost:5000
+          export KUBEVIRT_PROVIDER=external
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:211]: can be removed

**Comment by oshoval** [comment]
@maiqueb @qinqon 
if you want please review
it will be on hold until the two curl hacks are removed

**Review by maiqueb — APPROVED** [review]
Thank you.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -176,6 +176,69 @@ presubmits:
...
+          curl -L https://github.com/kubevirt/kubevirtci/pull/1193.patch | git am -3
+          curl -L https://github.com/kubevirt/kubevirt/pull/11410.patch | git am -3
+          export KUBEVIRT_PROVIDER=kind-ovn
+          make cluster-up
+          export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
+          make cluster-sync
+          export PATH=$(pwd)/_out/cmd/virtctl:$PATH
+          kubectl patch kubevirt -n kubevirt kubevirt --type merge --patch '{"spec":{"configuration":{"developerConfiguration":{"featureGates":["PersistentIPs"]}}}}'
+          cd _ci-configs/kind-ovn/_ovnk/test/e2e
+          go test --test.timeout=1h --test.v --ginkgo.v --ginkgo.noColor=true --ginkgo.focus="should keep ip"
```
Comment (maiqueb) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:216]: nit: 
```
  --ginkgo.noColor is deprecated, use --ginkgo.no-color instead
```

Comment (oshoval) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:216]: yep :) you are fast, waited for the lane to finish first

**Comment by oshoval** [comment]
addressed comments

**Comment by oshoval** [comment]
rebased

**Comment by oshoval** [comment]
lets focus on https://github.com/kubevirt/project-infra/pull/3433

**Pull request closed without merging**